b2sdk/transfer/inbound/downloaded_file.py

```diff
@@ -99,13 +99,13 @@ def __init__(
 
     def _validate_download(self, bytes_read, actual_sha1):
         if self.range_ is None:
-            if bytes_read != self.file_version.size:
-                raise TruncatedOutput(bytes_read, self.file_version.size)
+            if bytes_read != self.download_version.size:
+                raise TruncatedOutput(bytes_read, self.download_version.size)
```

Comment:
`content_length` - `size` is the size of file - and user might selected only a range...

Reply:
you are right. I did make the change in several places, this one must have slipped

Reply:
sorry - I read single commit and I missed the context :/